Sources/OpenAPIRuntime/Multipart/ByteUtilities.swift

```diff
@@ -0,0 +1,121 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftOpenAPIGenerator open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftOpenAPIGenerator project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftOpenAPIGenerator project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+/// A namespace of utilities for byte parsers and serializers.
+enum ASCII {
+
+    /// The dash `-` character.
+    static let dash: UInt8 = 0x2d
+
+    /// The carriage return `<CR>` character.
+    static let cr: UInt8 = 0x0d
+
+    /// The line feed `<LF>` character.
+    static let lf: UInt8 = 0x0a
+
+    /// The colon `:` character.
+    static let colon: UInt8 = 0x3a
+
+    /// The space ` ` character.
+    static let space: UInt8 = 0x20
+
+    /// The horizontal tab `<TAB>` character.
+    static let tab: UInt8 = 0x20
```

Comment:
```suggestion
    static let tab: UInt8 = 0x09
```

Reply:
Copy pasta FTL.

Thanks for catching that.